### PR TITLE
hand_Contours

### LINKED ISSUE #6: User's hand contours and centroid
- Draw the contours for the user's hand using the mask created based on the user's hand color.
- Also, locate the centroid of the hand.

### PR DESCRIPTION
**Pull request by singh-nicky**
draw contours for a hand

### CODE REVIEW

**Review by PK-cod3ch3mist — APPROVED** [review]
Looks good to me, @AmitSingh12345678 and @Sanchit-sk please review once.

**Review by AmitSingh12345678 — APPROVED** [review]

File: Gesture_control.py [hunk 1]
```diff
@@ -35,14 +35,28 @@ def create_mask(img):
     #cv2.imshow('Mask', mask)
     return mask
 
+def thresholding(mask):
+    _,thresh = cv2.threshold(mask,127,255,cv2.THRESH_BINARY) # if pixel intensity <= 127 then set it as 0 and pixel intensity > 127 set it as 255
+    return thresh
+
+def find_Contours(thresh):
```
Comment (Sanchit-sk) [inline Gesture_control.py]: 'C' should be in lowercase
Rename this function to find_contours

File: Gesture_control.py [hunk 2]
```diff
@@ -35,14 +35,28 @@ def create_mask(img):
     #cv2.imshow('Mask', mask)
     return mask
 
+def thresholding(mask):
```
Comment (Sanchit-sk) [inline Gesture_control.py]: It may be renamed to threshold rather than thresholding

**Review by Sanchit-sk — CHANGES_REQUESTED** [review]
Just make these minor changes.
Rest everything looks great!

**Comment by Sanchit-sk** [comment]
Also, complete the centroid formation part.
Be mindful of these things: -

- There are multiple contours even within the hand and also due to some objects in the background. Make sure to take the centroid of only the hand contour. (Hint: You might want to consider the largest contour).
- To remove the contours for the details of the hand (if coming), you can rather use RETR_EXTERNAL instead of RETR_TREE.
- Try experimenting with these and push these changes

**Comment by Sanchit-sk** [comment]
Everything seems good to me
Merging to main!

**Pull request merged**